src/fragment-components/tiles/a-radio-tile-group.tsx

```diff
@@ -151,29 +151,30 @@ export const componentInfo: ComponentInfo = {
 	codeUI: ARadioTileGroupCodeUI,
 	keywords: ['tile', 'card', 'radio', 'select'],
 	name: 'Radio tile group',
+	type: 'radio-tile-group',
 	defaultComponentObj: {
 		type: 'radioTileGroup',
```

Comment:
Shouldn't this be radio-tile-group?

```suggestion
		type: 'radio-tile-group',
```